servicetalk-http-netty/build.gradle

```diff
@@ -34,8 +34,9 @@ dependencies {
   implementation project(":servicetalk-transport-netty-internal")
   implementation project(":servicetalk-utils-internal")
   implementation "com.google.code.findbugs:jsr305:$jsr305Version"
-  implementation "io.netty:netty-codec-http:$nettyVersion"
-  implementation "io.netty:netty-codec-http2:$nettyVersion"
+  implementation(platform("io.netty:netty-bom:$nettyVersion"))
```

Comment:
I don't know if there is any best practice here, but perhaps putting the platform dependency at the top would provide a cleaner outcome?

Reply:
That is the way I started but then opted to move it to the other Netty lines to produce smaller diffs.

Now does seem like a good time to choose a convention. I would be fine with either "platforms at the top" or "platforms before first use".

Reply:
Now placing all BOM declarations at top of dependencies section.